ide-support/revliburl.livecodescript

```diff
@@ -73,6 +80,49 @@ on libraryStack
    initialiseCustomProps
 end libraryStack
 ------------------------------------------------
+-- CW-2016-06-11: [[ External driver support ]] Add function to select external driver for use with libUrl.
+## Set driver to use for libUrl
+on libUrlSetDriver pDriver
```

Comment:
@montegoulding The command should be documented to some degree at the very least.  I will add an lcdoc for it, though the details will be pretty minimal as the result of a change of driver will be specific to the type of driver in use.